### PR TITLE
Replace all usage of "osu!lazer"

### PR DESCRIPTION
**Pull request by peppy**
This is no longer correct terminology. "lazer" was always intended to be a codename and has already been dropped from all public facing usage in releases some time ago.

The `(lazer)` suffix will eventually be dropped when things go live, although that will need to be done alongside careful distinction of articles that are specific to the old client behaviour.  I am intentionally wanting to rewrite history here to avoid the name sticking, since it'll either be no longer relevant or replaced with a new release name in the future.

I've done a quick manual pass but some changes may need further discussion.

`SKIP_OUTDATED_CHECK` (`People/Users_with_unique_titles/{fr,id}.md`)

### CODE REVIEW

**Comment by Walavouchey** [comment]
This has been on the agenda for a while, but why `(lazer release)`? there's been discussion on the [relevant issue](https://github.com/ppy/osu-wiki/issues/6590) way more logical alternatives. Either way, this kind of suffix isn't best fit for every situation (while something like a temporary `osu!(lazer)` would fit more situations). `lazer release of osu!` or `lazer version of the osu! client` or `lazer version of osu!` would work way better in writing.

I'm *highly against* rewriting history here. Both for news posts and history articles. It's fine to replace and reword instances to provide more context (especially in the history articles), but I highly encourage leaving the older news posts as they are.

Correcting it in wiki articles and future news posts alone is enough, because the old posts wouldn't affect public perception due to almost no one going back to read them. The same would go for forum posts - there's no point in going back to those to change it.

I've taken some random instances and left reviews on them to illustrate what I mean, with some random but better alternatives. More input is of course required, but blindly attaching a suffix everywhere isn't a good solution. I will have to comb through this all in the weekend.

File: wiki/Tournament_drawings/en.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ This line has three sections separated by a colon (`:`):
 | :-: | :-: | :-: |
 | AU | Australia | AUS |
 
-- The flag name refers to the name of a file that provides the flag image. By default, osu!lazer provides country flags as [ISO 3166 Alpha-2 Country Codes](https://www.iso.org/iso-3166-country-codes.html).
+- The flag name refers to the name of a file that provides the flag image. By default, osu! (lazer release) provides country flags as [ISO 3166 Alpha-2 Country Codes](https://www.iso.org/iso-3166-country-codes.html).
```
[the remark this answers is not in the document]
Comment (peppy) [reply wiki/Tournament_drawings/en.md:84]: yeah, not having it written this way. the whole reason for standardising as per this PR is that we can easily drop it when the time comes to do so.

**Comment by peppy** [comment]
> This has been on the agenda for a while, but why `(lazer release)`? there's been discussion on the [relevant issue](https://github.com/ppy/osu-wiki/issues/6590) with way more logical alternatives. Either way, this kind of suffix isn't best fit for every situation (while something like a temporary `osu!(lazer)` would fit more situations). `lazer release of osu!` or `lazer version of the osu! client` or `lazer version of osu!` would work way better in writing.

We can use osu!(lazer) I think, as long as it's consistent. I felt that `(lazer release)` was more descriptive and gives people a point to search further, but can work with the former.

> 
> I'm _highly against_ rewriting history here. Both for news posts and history articles. It's fine to replace and reword instances to provide more context (especially in the history articles), but I highly encourage leaving the older news posts as they are.

I'd want to rewrite stuff like the spotlight posts at very least, which have had it written incorrectly forever. We can leave posts which describe the history from a perspective of introducing the original project or codename though.

> Correcting it in wiki articles and future news posts alone is enough, because the old posts wouldn't affect public perception due to almost no one going back to read them. The same would go for forum posts - there's no point in going back to those to change it.

I'd want to correct news posts as they are generally considered official, and have used the wrong language. Would prefer they don't get indexed by google, for instance.

**Comment by Walavouchey** [comment]
> We can use osu!(lazer) I think, as long as it's consistent. I felt that (lazer release) was more descriptive and gives people a point to search further, but can work with the former.

"(lazer release)" is way clunkier than it needs to be. Ideally we'd just reword all instances to fit the current standard and be done with it in one pr. The diffstat is about 250 (and even lower with old news posts unchanged) - going through everything can be done in one or two sessions.

> I'd want to rewrite stuff like the spotlight posts at very least, which have had it written incorrectly forever. We can leave posts which describe the history from a perspective of introducing the original project or codename though.

I'll tentatively say that correcting the spotlights news posts is fine, but I'd want to do it in a way that will stick going forward, such that no other change is made to them afterwards. Even when the majority of the userbase migrates over, it would still read as "the lazer version of osu!" in those posts, as they are still history after all.

> I'd want to correct news posts as they are generally considered official, and have used the wrong language. Would prefer they don't get indexed by google, for instance.

Updating news posts to use new terminology makes them not current to their post date anymore. They are "news", not wiki articles. You post it, perhaps amend it, then leave it. That's how all news publications work, official or not.

Sure, if you were to search "osu!lazer" when all is said and done, you'd find old news articles and forum posts. With all current articles using correct terminology and with more people using lazer, people will likely start to drop "osu!lazer" eventually anyway.

Whether people will still use "osu!lazer" is outside anyone's control, so a blank rename in every news post is pointless because it wouldn't remove it from public conscience. The same goes for people referring to osu! as "Osu!" and the osu! game mode as "osu!std".

File: wiki/Beatmap_Spotlights/Seasons/2022_Winter/en.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ As of now, joining the weekly multiplayer lobbies requires participants to downl
 3. Within a playlist lobby, everyone competes towards a room-specific leaderboard.
    - Only the best score results on each beatmap within the playlist are summed up to create a weekly ranked score for each player.
    - Each beatmap of a playlist counts equally to the leaderboard score.
-   - The individual beatmap score is based on osu!lazer's scoring system.
+   - The individual beatmap score is based on osu! (lazer release)'s scoring system.
```
[the remark this answers is not in the document]
Comment (peppy) [reply wiki/Beatmap_Spotlights/Seasons/2022_Winter/en.md]: I think this can stay as-is now.

**Comment by Walavouchey** [comment]
I've pushed some changes that reorder the tournament client articles as I [mentioned on Discord](https://discord.com/channels/188630481301012481/218677502141399041/944142591836979240). I think it should be pretty easy to grasp by looking at [d2e56eb](https://github.com/ppy/osu-wiki/pull/6888/commits/d2e56eb3bceeff2a503266a60c28b41f1de3509b) and [6cadc8a](https://github.com/ppy/osu-wiki/pull/6888/commits/6cadc8a931108f84fc5d1d515a0a28d08a3ae018).

Pretty sure this change alone should be enough for now, but I'll also look at the history articles in a bit, because I think it would be good to get those in a good state right away

File: wiki/History_of_osu!/en.md [hunk 3]
```diff
@@ -16,8 +16,8 @@ Records of osu!'s history, all in the osu! wiki.
 - [2013](/wiki/History_of_osu!/2013) - UI overhaul, HD skins, improved widescreen support, osu!mania editor, in-game collections, moddingV2 was introduced, Disqus, osu!api, accuracy overlays, Comic Fiesta 2013, downloads unrestricted, and osu!cuttingedge (osu!supporters only)
 - [2014](/wiki/History_of_osu!/2014) - BPM scrolling, in-game intro/outro sequences, osu!cuttingedge (for everyone), and snow!
 - [2015](/wiki/History_of_osu!/2015) - jizz (redesigned osu!web), osu!weekly, and osu!coins
-- [2016](/wiki/History_of_osu!/2016) - scorev2, open-source osu! announced (osu!lazer), and osu! wiki (overhaul)
-- [2017](/wiki/History_of_osu!/2017) - first osu!lazer build release, and Discord integration
+- [2016](/wiki/History_of_osu!/2016) - scorev2, open-source osu! announced (osu!(lazer)), and osu! wiki (overhaul)
+- [2017](/wiki/History_of_osu!/2017) - first osu!(lazer) build release, and Discord integration
```
Comment (Walavouchey) [inline wiki/History_of_osu!/en.md:20]:
```suggestion
- [2017](/wiki/History_of_osu!/2017) - release of the first lazer build of osu!, and Discord integration on stable
```

File: wiki/History_of_osu!/2016/en.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ Links:
 
 ## June
 
-Mapping With Rewards \#1, announced in the previous year, has finally been concluded. osu!idol 2016 opened. peppy announced osu!lazer, the soon-to-be successor of the current osu! client.
+Mapping With Rewards \#1, announced in the previous year, has finally been concluded. osu!idol 2016 opened. peppy announced osu!(lazer), the soon-to-be successor of the current osu! client.
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2016/en.md:47]:
```suggestion
Mapping With Rewards \#1, announced in the previous year, has finally been concluded. osu!idol 2016 opened. peppy announced a new osu! client, codenamed "lazer" (previously known as "osu!lazer"), the soon-to-be successor of the current osu! client.
```

File: wiki/History_of_osu!/2018/en.md [hunk 5]
```diff
@@ -10,13 +10,13 @@ After a six-month long hiatus, flyte posted on the [osu!next](https://osunext.tu
...
+In osu!(lazer), background blurring and the hidden mod were added. osu!mania scrolling speed was implemented. Thanks to cYsmix, a fresh set of default hitsounds was added. osu!catch was vastly improved and autoplay was fixed to be more human-like.
 
 In osu!web, beatmap pack listings were improved, hype and nomination counters were visible to the beatmap listings. Obtaining a permalink to beatmap discussion post was made easier to copy, localisations can now resize the menu upon hovering over the locales, star ratings can be seen on the beatmap listing via hovering the difficulty icon, and sorting of beatmap discussions was added. A limit to only load 250 forum threads and unload others was implemented to avoid heavy memory consumption.
 
 peppy tweeted that the new t-shirt inventory (along with their new designs) has arrived. Six days later, they were up on the osu!store.
 
-The ability to see star ratings in the osu!direct was backported from osu!lazer. Lastly, thanks to nekodex, the scoreboard redesign was implemented.
+The ability to see star ratings in the osu!direct was backported from osu!(lazer). Lastly, thanks to nekodex, the scoreboard redesign was implemented.
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2018/en.md:19]:
```suggestion
The ability to see star ratings in the osu!direct was backported from the lazer version of the osu! client. Lastly, thanks to nekodex, the scoreboard redesign was implemented.
```

File: wiki/History_of_osu!/2017/en.md [hunk 6]
```diff
@@ -121,7 +121,7 @@ Links:
 
 ## December
 
-The Winter 2017 Fanart Contest results were in, awarding [Leissss](https://osu.ppy.sh/users/4750716) with the most votes. OWC 2017 concluded with Poland being victorious. Thanks to LiquidPL, the cringey download page in osu!web has been redone. The day before Christmas, another handful of community contributors were announced. Every player received osu!supporter status temporarily for testing purposes, which coincidentally happened on Christmas Eve. peppy ends the year with a video, giving 2017 a year-in-review in regards to osu!lazer.
+The Winter 2017 Fanart Contest results were in, awarding [Leissss](https://osu.ppy.sh/users/4750716) with the most votes. OWC 2017 concluded with Poland being victorious. Thanks to LiquidPL, the cringey download page in osu!web has been redone. The day before Christmas, another handful of community contributors were announced. Every player received osu!supporter status temporarily for testing purposes, which coincidentally happened on Christmas Eve. peppy ends the year with a video, giving 2017 a year-in-review in regards to osu!(lazer).
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2017/en.md:124]:
```suggestion
The Winter 2017 Fanart Contest results were in, awarding [Leissss](https://osu.ppy.sh/users/4750716) with the most votes. OWC 2017 concluded with Poland being victorious. Thanks to LiquidPL, the cringey download page in osu!web has been redone. The day before Christmas, another handful of community contributors were announced. Every player received osu!supporter status temporarily for testing purposes, which coincidentally happened on Christmas Eve. peppy ends the year with a video, giving 2017 a year-in-review in regards to the lazer version of osu!.
```

File: wiki/History_of_osu!/2016/en.md [hunk 7]
```diff
@@ -86,7 +86,7 @@ Links:
...
+osu! World Cup 2016 registrations opened. GML applications were opened for those who wish to join. The [loved](/wiki/Beatmap/Category#loved) beatmap status was introduced in its testing stage as a way to give community-loved, but unrankable, beatmaps leaderboards. Those types of beatmaps will, however, not award [pp](/wiki/Performance_points) and their leaderboards could be deleted for when the creator updates their beatmap. The Halloween-themed 7th fanart contest opened. Loved was in its second testing state, ready for a new batch of beatmaps to enter this category. A new video featuring osu!(lazer) was released. Most notably, the hit circles had more textures to them and gave off a smooth and shiny look. osu!idol 2016 voting were opened. A few days before Halloween, the Halloween fanart contest was opened and concluded two days later. The osu!weekly released the 78th issue on Halloween, making it as spooky as possible (but where did the 77th issue go? We'll never know).
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2016/en.md:89]:
```suggestion
osu! World Cup 2016 registrations opened. GML applications were opened for those who wish to join. The [loved](/wiki/Beatmap/Category#loved) beatmap status was introduced in its testing stage as a way to give community-loved, but unrankable, beatmaps leaderboards. Those types of beatmaps will, however, not award [pp](/wiki/Performance_points) and their leaderboards could be deleted for when the creator updates their beatmap. The Halloween-themed 7th fanart contest opened. Loved was in its second testing state, ready for a new batch of beatmaps to enter this category. A new video featuring the new osu! client (lazer) was released. Most notably, the hit circles had more textures to them and gave off a smooth and shiny look. osu!idol 2016 voting were opened. A few days before Halloween, the Halloween fanart contest was opened and concluded two days later. The osu!weekly released the 78th issue on Halloween, making it as spooky as possible (but where did the 77th issue go? We'll never know).
```

File: wiki/History_of_osu!/2017/en.md [hunk 8]
```diff
@@ -12,7 +12,7 @@ Links:
 
 ## February
 
-[Ben Briggs](https://osu.ppy.sh/beatmaps/artists/11) was announced to be another Featured Artist. The first osu!lazer build was released!
+[Ben Briggs](https://osu.ppy.sh/beatmaps/artists/11) was announced to be another Featured Artist. The first osu!(lazer) build was released!
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2017/en.md:15]:
```suggestion
[Ben Briggs](https://osu.ppy.sh/beatmaps/artists/11) was announced to be another Featured Artist. The first lazer build of osu! was released!
```

File: wiki/History_of_osu!/2018/en.md [hunk 9]
```diff
@@ -68,7 +68,7 @@ Links:
 
 Trial & Error was introduced as a new Featured Artist.
 
-In osu!lazer, basic skinning support was implemented. The parallax effect was reduced during gameplay.
+In osu!(lazer), basic skinning support was implemented. The parallax effect was reduced during gameplay.
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2018/en.md:71]:
```suggestion
In the lazer version of osu!, basic skinning support was implemented. The parallax effect was reduced during gameplay.
```

File: wiki/History_of_osu!/April_Fools/en.md [hunk 10]
```diff
@@ -165,7 +165,7 @@ As part of the handful of April Fools' day jokes for 2016, the [osu! cookie](/wi
 
 ## 2017
 
-As announced in a [tweet by peppy](https://twitter.com/ppy/status/848021525663842304), there was no April Fools' day joke for osu! in 2017 due to osu!lazer development.
+As announced in a [tweet by peppy](https://twitter.com/ppy/status/848021525663842304), there was no April Fools' day joke for osu! in 2017 due to osu!(lazer) development.
```
Comment (Walavouchey) [inline wiki/History_of_osu!/April_Fools/en.md:168]:
```suggestion
As announced in a [tweet by peppy](https://twitter.com/ppy/status/848021525663842304), there was no April Fools' day joke for osu! in 2017 due to development on the lazer version of the osu! client.
```

File: wiki/osu!_tournament_client/en.md [hunk 11]
```diff
@@ -1,27 +1,27 @@
...
+The **osu! tournament client** is the official client that overlays [osu!tourney](/wiki/osu!tourney) with supplementary scenes and information relevant to an osu! tournament during a live stream.
 
 Users encountering problems with the client may [create an issue on GitHub](https://github.com/ppy/osu/issues) or send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
 
 ## Setup
 
-To start the osu!lazer tournament client, you need to specify a launch argument to the osu!lazer executable. To do this, create a new shortcut on the desktop and set the location for it to `%LOCALAPPDATA%/osulazer/osu!.exe --tournament`. This will make that specific shortcut start up osu!lazer in the tournament client mode.
+To start the osu! tournament client, you need to specify a launch argument to the osu!(lazer) executable. To do this, create a new shortcut on the desktop and set the location for it to `%LOCALAPPDATA%/osulazer/osu!.exe --tournament`. This will make that specific shortcut start up osu!(lazer) in the tournament client mode.
```
Comment (Walavouchey) [inline wiki/osu!_tournament_client/en.md:9]: > `%LOCALAPPDATA%/osulazer/`

:|

**Review by Walavouchey — CHANGES_REQUESTED** [review]
some additional outdating will be needed after applying these

the reviews are a little out of order, so i suggest viewing them in the "files changed" tab

File: wiki/osu!_tournament_client/en.md (hunk 11 above)
Comment (peppy) [reply wiki/osu!_tournament_client/en.md:9]: This is a bit of an unfortunate one yeah. `osu!` is already taken so we can't reuse it here.

Parallels Desktop 2022-02-21 at 07 26 31

It's on my list of things we need to revisit alongside improving the migration and update process.

File: wiki/History_of_osu!/2018/en.md (hunk 5 above)
Comment (peppy) [reply wiki/History_of_osu!/2018/en.md:19]: Not 100% sure which of these reads better (strictly speaking "lazer" is not a version but a release stream), but let's go with what you have for now.

File: wiki/History_of_osu!/2018/en.md [hunk 12]
```diff
@@ -10,13 +10,13 @@ After a six-month long hiatus, flyte posted on the [osu!next](https://osunext.tu
 
 The osu!mania 7K World Cup 2018 registrations opened. Community Choice 2017 (previously known as the "Best of" series) voting was opened.
 
-In osu!lazer, background blurring and the hidden mod were added. osu!mania scrolling speed was implemented. Thanks to cYsmix, a fresh set of default hitsounds was added. osu!catch was vastly improved and autoplay was fixed to be more human-like.
+In osu!(lazer), background blurring and the hidden mod were added. osu!mania scrolling speed was implemented. Thanks to cYsmix, a fresh set of default hitsounds was added. osu!catch was vastly improved and autoplay was fixed to be more human-like.
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2018/en.md:13]: missed this one

```suggestion
In the lazer version of osu!, background blurring and the hidden mod were added. osu!mania scrolling speed was implemented. Thanks to cYsmix, a fresh set of default hitsounds was added. osu!catch was vastly improved and autoplay was fixed to be more human-like.
```

**Review by Walavouchey — APPROVED** [review]
looks alright now

**Pull request merged**